packages/kbn-shared-ux-components/src/page_template/no_data_page/types.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { CommonProps, EuiCardProps } from '@elastic/eui';
+import { MouseEventHandler, ReactNode } from 'react';
+
+export type NoDataPageActions = Partial<EuiCardProps> & {
+  /**
+   * Provide just a string for the button's label, or a whole component
+   */
+  button?: string | ReactNode;
+  /**
+   * Remapping `onClick` to any element
+   */
+  onClick?: MouseEventHandler<HTMLElement>;
+  /**
+   * Category to auto-select within Fleet
+   */
+  category?: string;
+};
```

Comment:
These are actually extending `ElasticAgentCardProps` without the `description`. Maybe we can just extend otherwise we'll need to keep the props and descriptions in sync?

```suggestion
export type NoDataPageActions = Omit<ElasticAgentCardProps, 'description'>;
```

Reply:
Fixed!